### ISSUE TITLE
scriptEl.parentNode appears to be null when executing Phantom JS and IE tests involving sails.io

### ISSUE THREAD

**Issue by fristys** [balderdashy/sails.io.js#63]
I'm using Karma / Jasmine to run my front end tests that involve sails.io as part of my dependencies and I seem to be getting false positive failed tests, or even non-executed tests in IE / Phantom JS, because of ``scriptEl.parentNode`` being `null` instead of an ``object`` in ``dist/sails.io.js`` line 207 column 9 when running the tests (if I open the app in IE myself it doesn't produce this error).

I came up with a simple solution that seems to fix the problem and decided to share so it might be looked at and possibly added. Replacing the said line (207) with this one fixes the issue and doesn't interfere with the sails.io operations in my normal application as far as I can tell:

```javascript
if (scriptEl && scriptEl.parentNode) scriptEl.parentNode.removeChild(scriptEl);
```

**Comment by sailsbot** [comment]
Thanks for posting, @fristys.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by rkcreation** [comment]
+1

**Comment by sgress454** [comment]
Fixed by #93
